client/js/router/index.js

```diff
@@ -11,12 +11,17 @@ import store from '../store/store.js';
 Vue.use(VueRouter);
 
 const mustBeAuthenticated = (to, from, next) => {
-  if (store.state.auth.authenticated) {
+  if (store.getters.authenticated) {
     return next();
   }
   return next('/login');
 };
 
+const logoutAndRedirect = async (to, from, next) => {
+  await store.dispatch('requestLogout');
+  return next('/login');
+};
```

Comment:
Middleware that will run anytime we hit `/logout` from Vue. This will ensure that sails cleans up the session and we properly redirect the user to the login page.